Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs

```diff
@@ -202,6 +199,12 @@ static void UpdateCSC()
                 "649"
             };
 
+            if(!File.Exists(cscFilePath))
+            {
+                Debug.LogFormat("csc.rsp not found at the top level directory: {0}, no modifications were made", cscFilePath);
```

Comment:
Do you think this should be a silent return? It feels like this might be an okay (and probably the most common) case, and seeing this log _could_ be confusing (do I need a csc.rsp? why don't i have one?)

Reply:
Yeah you're right making this silent should reduce the confusion for people importing our pacakge.